strings/loot_data/tags/item.txt

```diff
@@ -1990,7 +1990,8 @@
 /obj/item/weapon/gun_upgrade/trigger/boom    blacklisted=1    spawn_value=10    price_tag=100   100    
 /obj/item/weapon/gun_upgrade/mechanism/overshooter    blacklisted=0    spawn_value=5    price_tag=100   100    
 /obj/item/weapon/gun_upgrade/mechanism/glass_widow    blacklisted=1    spawn_value=2    price_tag=100   100    
-/obj/item/weapon/gun_upgrade/mechanism/weintraub    blacklisted=0    spawn_value=3.33333    price_tag=100   100    
+/obj/item/weapon/gun_upgrade/mechanism/weintraub    blacklisted=0    spawn_value=3.33333    price_tag=100   100
```

Comment:
This is an automatically generated file and doesn't need to be changed for this PR.

Reply:
Should hopefully be removed from the PR.